i18n/extra/SemanticMediaWiki.magic.php

```diff
@@ -386,16 +386,16 @@
 /** Simplified Chinese (中文（简体）‎) */
 $magicWords['zh-hans'] = [
 	'ask' => [ 0, '询问' ],
-	'show' => [ 0, '显示' ],
+	'show' => [ 0, '展示', '显示' ],
 	'info' => [ 0, '信息' ],
 	'concept' => [ 0, '概念' ],
 	'subobject' => [ 0, '子对象' ],
-	'smwdoc' => [ 0, 'SMW文档' ],
+	'smwdoc' => [ 0, 'smw文档', 'SMW文档' ],
```

Comment:
Magic words aliases are case insensitive.

Reply:
You're right. I will fix this.

Reply:
(mark as resolved)